### DISCUSSION TITLE
Beta 3. Problem in js bundled

### DISCUSSION THREAD

**Discussion by abkrim** [livewire/livewire#6277] [7. Bugs (archived)]
### Livewire version

v3-beta

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. USe with Filamenadmin
2. Login page
3. This show a problem in livewire.js

incorrect livewire.js at the end (error on comments ?)
```
/*!
* focus-trap 6.6.1
* @license MIT, https://github.com/focus-trap/focus-trap/blob/master/LICENSE
*/
/*!
* tabbable 5.2.1
* @license MIT, https://github.com/focus-trap/tabbable/blob/master/LICENSE
*/
/*! Bundled license information:   

nprogress/nprogress.js:
  (* NProgress, (c) 2013, 2014 Rico Sta. Cruz - http://ricostacruz.com/nprogress
   * @license MIT *)
   ```
   
![CastrisLogin - Swissknife0808210954](https://github.com/livewire/livewire/assets/1238625/25c104d3-6e7e-462d-b280-96743666440f)

   

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PepinoCuellar** [comment]
Did you find out how to solve it?
I've got the same problem

**Comment by calebporzio** [comment]
Hmmm I haven't been able to reproduce this - can you provide more guidance for others to reproduce so they can help fix it?
